src/ui/components/button/Button.module.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  padding: 10px 16px;
+  font-size: 14px;
+  line-height: 17px;
+  background-color: rgb(255, 153, 51);
+  border: 1px solid rgb(255, 128, 0);
+  font-weight: 600;
+  color: white;
```

Comment:
Use css variables (and dont just call it white), this project should never use colors by names or hex codes directly